src/components/SuperDrawer/ConfirmCloseModal.tsx

```diff
@@ -23,7 +23,10 @@ export function ConfirmCloseModal({ onClose }: ConfirmCloseModalProps) {
       <p css={Css.base.gray700.$}>Any changes you've made so far will be lost.</p>
       <div css={Css.mt4.gap1.$}>
         <Button variant="tertiary" label="Cancel" onClick={closeModal} />
-        <Button label="Close" onClick={onClose} />
+        <Button label="Close" onClick={() => {
+          onClose();
+          closeModal();
```

Comment:
I wonder if the order of `closeModal` vs. `onClose` matters here? Probably not?

Either way, maybe throw in a comment like:

```
// Order doesn't really matter; close this modal + tell the caller to actually do their close
```

I know this isn't directly part of your PR, but just to clarify some of this code, wdyt about while you're here renaming `ConfirmCloseModalProps.onClose` to `ConfirmCloseModalProps.onConfirm`?

Reply:
Another thing is that most of our drawer closing logic does include closing the modal. Could we just assume that?

Reply:
@KoltonG are you saying we shouldn't rename the props?

Reply:
Sorry! I meant that most, maybe all, of the `onClose` functions (the ones in closeDrawer and closeDrawerDetail) that are passed to the "Confirmation Modal" when a check fails includes logic to remove the modalContent via `modalContent.current = null`. 

If that is the case, just wondering if we still need to be explicit with `closeModal()`? I'm not sure if I missed why we included this.

Reply:
Ahh I see. The doChange function that openInDrawer pases didn't explicitly touch the modalContent. I suppose we could change that, though?

Reply:
Got it! Alright if it didn't handle that then let's keep this but make sure we don't have `modalContent.current = null` in the lambda being passed in.